packages/mendel-browserify/index.js

```diff
@@ -203,6 +210,35 @@ MendelBrowserify.prototype.doneManifest = function() {
     );
 }
 
+MendelBrowserify.prototype.sortedManifest = function() {
+    var sortedManifest = {
+        indexes: {},
+        bundles: []
+    };
+
+    var self = this;
+    Object.keys(this._manifestIndexes).sort().forEach(function(file) {
+        var bundle = self._manifestBundles[self._manifestIndexes[file]];
+        sortedManifest.bundles.push(bundle);
+
+        var index = sortedManifest.bundles.indexOf(bundle);
```

Comment:
WAT? It is working, if it was wrong it would totally break the tests and examples. Besides, up on the file there is a very similar line and it is the same for ages.